subcommands/keys/tuf_updates_rotate_offline_key.go

```diff
@@ -185,16 +185,20 @@ func doTufUpdatesRotateOfflineTargetsKey(cmd *cobra.Command) {
 			subcommands.SliceRemove(curCiRoot.Signed.Roles["targets"].KeyIDs, onlineTargetsId))
 		subcommands.DieNotNil(err)
 	}
-	newTargetsSigs, err := signProdTargets(factory, newKey,
-		func(tag string, targets client.AtsTufTargets) bool {
```

Comment:
`tag` wasn't used in the previous version at all.

Reply:
A `tag` was used indirectly in this code block inside `signProdTargets` (see below):
```
for tag, targets := range targetsMap {
		if excludeFunc != nil && excludeFunc(tag, targets) {
			continue
		}
```

That logic was rather convoluted and really hard to follow.
So, that code is now gone, and that loop was replicated here.

It is kind of those problems when a _complicated code just works_ until a more complex problem arrives which desperately calls for simplification.